code/modules/bitrunning/objects/gimmick_disks/dungeon_disk.dm

```diff
@@ -0,0 +1,222 @@
+ */
+/obj/item/bitrunning_disk/gimmick/dungeon
+	name = "bitrunning gimmick: dungeon crawling"
+	selectable_loadouts = list(
+		/datum/bitrunning_gimmick/alchemist,
+		/datum/bitrunning_gimmick/rogue,
+		/datum/bitrunning_gimmick/healer,
+		/datum/bitrunning_gimmick/wizard,
+	)
+
+
+/datum/bitrunning_gimmick/alchemist
+	name = "Alchemist"
+
+	granted_items = list(
+		/obj/item/clothing/suit/bio_suit/plaguedoctorsuit,
+		/obj/item/clothing/mask/gas/plaguedoctor,
+		/obj/item/clothing/head/bio_hood/plague,
+		/obj/item/storage/box/alchemist_basic_chems,
+		/obj/item/storage/box/alchemist_basic_chems,
+		/obj/item/storage/box/alchemist_random_chems,
+		/obj/item/storage/box/alchemist_chemistry_kit,
+	)
+
+/obj/item/reagent_containers/cup/bottle/alchemist_basic
+	name = "unlabeled bottle"
+	desc = "A small bottle. You don't remember what you put in it."
+
+	/// List of possible reagents we may pick from
+	var/static/list/possible_reagents = list(
+		/datum/reagent/aluminium, // Basic chems
+		/datum/reagent/bromine,
+		/datum/reagent/carbon,
+		/datum/reagent/chlorine,
+		/datum/reagent/copper,
+		/datum/reagent/consumable/ethanol,
+		/datum/reagent/fluorine,
+		/datum/reagent/hydrogen,
+		/datum/reagent/iodine,
+		/datum/reagent/iron,
+		/datum/reagent/lithium,
+		/datum/reagent/mercury,
+		/datum/reagent/nitrogen,
+		/datum/reagent/oxygen,
+		/datum/reagent/phosphorus,
+		/datum/reagent/potassium,
+		/datum/reagent/uranium/radium,
+		/datum/reagent/silicon,
+		/datum/reagent/sodium,
+		/datum/reagent/stable_plasma,
+		/datum/reagent/consumable/sugar,
+		/datum/reagent/sulfur,
+		/datum/reagent/toxin/acid,
+		/datum/reagent/water,
+		/datum/reagent/fuel,
+		/datum/reagent/toxin/leadacetate, // Funny chems
+		/datum/reagent/consumable/liquidgibs,
+		/datum/reagent/consumable/nutriment,
+		/datum/reagent/consumable/coffee,
+	)
+
+/obj/item/reagent_containers/cup/bottle/alchemist_basic/Initialize(mapload)
+	list_reagents = list(pick(possible_reagents) = 50)
+	. = ..()
+
+/obj/item/reagent_containers/cup/bottle/alchemist_random
+	name = "skull-labeled bottle"
+	desc = "A small bottle. You don't remember what you put in it."
+
+	/// List of random names this may have
+	var/static/list/possible_names = list(
+		"unlabeled bottle",
+		"skull-labeled bottle", // Labels
+		"heart-labeled bottle",
+		"explosion-labeled bottle",
+		"fish-labeled bottle",
+		"smiley-labeled bottle",
+		"frown-labeled bottle",
+		"interrobang-labeled bottle",
+		"plus-labeled bottle",
+		"d20-labeled bottle",
+		"unreadably-labeled bottle",
+		"black-labeled bottle",
+		"empty-labeled bottle",
+		"age-yellowed bottle", // Qualities
+		"blood-tinged bottle",
+		"ash-marred bottle",
+		"claw-scratched bottle",
+		"marker-marked bottle",
+		"cracked bottle",
+		"ominous bottle", // Abstract
+	)
+
+/obj/item/reagent_containers/cup/bottle/alchemist_random/Initialize(mapload)
+	list_reagents = list(get_random_reagent_id() = rand(20, 50))
```

Comment:
does this work byond stringifies list args pretty often

Reply:
works on my machine (yes, I tested it)
I can change it if you'd prefer

Reply:
I'd like that ye